src/shared/chainConfig/sepolia.ts

```diff
@@ -100,8 +100,8 @@ export let versionOverrideSepoliaMock: any = {
   },
   v2: {
     Addresses: {
-      Axiom: "0xF57252Fc4ff36D8d10B0b83d8272020D2B8eDd55",
-      AxiomQuery: "0x56A380e544606ad713A8A19BAc1903aa7C2017FF",
+      Axiom: "0x69963768F8407dE501029680dE46945F838Fc98B",
```

Comment:
Doesn't need to block this merge, but probably safer to call these `AxiomV2CoreMock` and `AxiomV2QueryMock`